Multisig/UI/Safe Management/Create Safe/AppleWeb3AuthLogin.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+import AuthenticationServices
+import CustomAuth
+import JWTDecode
+import UIKit
+
+class AppleWeb3AuthLogin: NSObject {
+    var onClose: () -> Void
+
+    init(onClose: @escaping () -> Void) {
+        self.onClose = onClose
+    }
+}
+
+extension AppleWeb3AuthLogin: ASAuthorizationControllerDelegate {
```

Comment:
do I understand correctly that this is not yet ready for the review? I see many debug statements, commented out code, and comments that look like pasted from a demo code